<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frontviewdialog_zhongcan_diancai_Dialog</class>
 <widget class="QDialog" name="frontviewdialog_zhongcan_diancai_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>607</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>250</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>500</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame#frame{
border: 2px  solid rgb(141, 92, 48);
border-radius:15px;
background:white;
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QFrame" name="frame_deskinfo">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QFrame#frame_deskinfo{
border: 2px  solid rgb(141, 92, 48);
border-radius:15px;
background:rgb(249,253,167);
}</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="leftMargin">
          <number>9</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>9</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="horizontalSpacing">
          <number>5</number>
         </property>
         <property name="verticalSpacing">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>开台时间</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>开始单号</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>桌号</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="label_deskno">
           <property name="text">
            <string>002</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLabel" name="label_17">
           <property name="text">
            <string>开台人</string>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QLabel" name="label_openperson">
           <property name="text">
            <string>XX</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>人数</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLabel" name="label_deskperson">
           <property name="text">
            <string>2</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QLabel" name="label_19">
           <property name="text">
            <string>服务员</string>
           </property>
          </widget>
         </item>
         <item row="3" column="3">
          <widget class="QLabel" name="label_deskoperid">
           <property name="text">
            <string>XXX</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="3">
          <widget class="QLabel" name="label_opemtime">
           <property name="text">
            <string>2015-05-25 14:23:58</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" colspan="3">
          <widget class="QLabel" name="label_openbill">
           <property name="text">
            <string>A08456132456</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="frontviewdialog_frame_list_item" name="frame_title">
       </widget>
      </item>
      <item>
       <widget class="frontViewDialog_list" name="listWidget">
        <property name="styleSheet">
         <string notr="true">QListWidget{
border:0px;
}

 QListWidget::item {
background:rgb(249,253,167);
border-bottom:3px solid white;
 }
 QListWidget::item:selected {
background-color: rgb(255, 139, 8);
 }</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>10</number>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
        <item>
         <widget class="QLabel" name="label_num">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(92, 86, 70);
color:white;
border-top-left-radius:10;
border-bottom-left-radius:10;
padding-left: 10px;
font-size:10pt;


</string>
          </property>
          <property name="text">
           <string>数量3</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_total">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(92, 86, 70);
color:white;
border-top-right-radius:10;
border-bottom-right-radius:10;
padding-right: 10px;
font-size:10pt;
</string>
          </property>
          <property name="text">
           <string>总计56</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_3">
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>0</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="pushButton_changyong">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>常用菜</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_shuliang">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>数量</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_jia">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_jian">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_cook">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>做法</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_changedish">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>换菜</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_cuicai">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>催菜</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_alldesk_cuicai">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>整桌催菜</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_del">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>删菜</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_timedish">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>临时菜</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_switch_desk">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>转台</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_other">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>其他</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4" stretch="1,0">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QFrame" name="frame_2">
       <property name="maximumSize">
        <size>
         <width>420</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QFrame#frame_2{
border: 2px  solid rgb(141, 92, 48);
border-radius:15px;
}</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,0,1,0">
        <item>
         <widget class="frontViewDialog_autoexpand" name="widget"/>
        </item>
        <item>
         <widget class="frontViewDialog_autoexpand" name="widget_2"/>
        </item>
        <item>
         <widget class="frontViewDialog_autoexpand" name="widget_3"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="pushButton_yuding">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>60</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>预订单</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>:/example/yudingdan.png</normaloff>:/example/yudingdan.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_bianma">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>60</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>编码点菜</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>:/example/bianma.png</normaloff>:/example/bianma.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_luodan">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>60</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>落单</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>:/example/luodan.png</normaloff>:/example/luodan.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_jiesuan">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>60</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>结算</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>:/example/jiesuan.png</normaloff>:/example/jiesuan.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_tuichu">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>60</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>退出</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>:/example/tuichu.png</normaloff>:/example/tuichu.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget_other_menu" native="true">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QPushButton" name="pushButton_discount">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
background-color: rgb(251, 6, 116);
border:0px; 
border-radius:0;
font-weight:bold;
}
QPushButton:pressed{
background-color: rgb(92, 86, 70);
}
QPushButton:default{
background-color: rgb(92, 86, 70);
}</string>
          </property>
          <property name="text">
           <string>折扣</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="pushButton_present">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
background-color: rgb(251, 6, 116);
border:0px; 
border-radius:0;
font-weight:bold;
}
QPushButton:pressed{
background-color: rgb(92, 86, 70);
}
QPushButton:default{
background-color: rgb(92, 86, 70);
}</string>
          </property>
          <property name="text">
           <string>赠送</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="pushButton_dish_switch_desk">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
background-color: rgb(251, 6, 116);
border:0px; 
border-radius:0;
font-weight:bold;
}
QPushButton:pressed{
background-color: rgb(92, 86, 70);
}
QPushButton:default{
background-color: rgb(92, 86, 70);
}</string>
          </property>
          <property name="text">
           <string>菜品换台</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="pushButton_cancel_desk">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
background-color: rgb(251, 6, 116);
border:0px; 
border-radius:0;
font-weight:bold;
}
QPushButton:pressed{
background-color: rgb(92, 86, 70);
}
QPushButton:default{
background-color: rgb(92, 86, 70);
}</string>
          </property>
          <property name="text">
           <string>消台</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="pushButton_desk_info">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
background-color: rgb(251, 6, 116);
border:0px; 
border-radius:0;
font-weight:bold;
}
QPushButton:pressed{
background-color: rgb(92, 86, 70);
}
QPushButton:default{
background-color: rgb(92, 86, 70);
}</string>
          </property>
          <property name="text">
           <string>开台信息</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>frontViewDialog_autoexpand</class>
   <extends>QFrame</extends>
   <header>frontviewdialog_autoexpand.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>frontViewDialog_list</class>
   <extends>QListWidget</extends>
   <header>frontviewdialog_list.h</header>
  </customwidget>
  <customwidget>
   <class>frontviewdialog_frame_list_item</class>
   <extends>QFrame</extends>
   <header>frontviewdialog_frame_list_item.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
